.env.template

```diff
@@ -31,3 +31,25 @@ BUCKET_REGION=eu-central-3
 BUCKET_ACCESS_KEY_ID=
 BUCKET_SECRET_ACCESS_KEY=
 MEDIA_BASE_URL=https://editor.serlo.dev #TODO: should be staging later
+
+# LTI platform: itslearning (staging)
+ITSLEARNING_URL=
+ITSLEARNING_NAME=
+ITSLEARNING_AUTHENTICATION_ENDPOINT="$ITSLEARNING_URL/connect/authorize"
+ITSLEARNING_ACCESS_TOKEN_ENDPOINT="$ITSLEARNING_URL/connect/token"
+ITSLEARNING_KEYSET_ENDPOINT="$ITSLEARNING_URL/.well-known/openid-configuration/jwks"
+SERLO_EDITOR_CLIENT_ID_ON_ITSLEARNING=
+
+# LTI platform: edu-sharing (RLP) (staging)
+EDUSHARING_RLP_URL=
+EDUSHARING_RLP_NAME="$EDUSHARING_RLP_URL"
+EDUSHARING_RLP_AUTHENTICATION_ENDPOINT="$EDUSHARING_RLP_URL/rest/ltiplatform/v13/auth"
+EDUSHARING_RLP_ACCESS_TOKEN_ENDPOINT="$EDUSHARING_RLP_URL/rest/ltiplatform/v13/token"
+EDUSHARING_RLP_KEYSET_ENDPOINT="$EDUSHARING_RLP_URL/rest/lti/v13/jwks"
+SERLO_EDITOR_CLIENT_ID_ON_EDUSHARING_RLP=
+
+# LTI tool: edu-sharing (RLP) (staging)
+EDUSHARING_RLP_LOGIN_ENDPOINT="$EDUSHARING_RLP_URL/rest/lti/v13/oidc/login_initiations"
+EDUSHARING_RLP_LAUNCH_ENDPOINT="$EDUSHARING_RLP_URL/rest/lti/v13/lti13"
+EDUSHARING_RLP_DETAILS_ENDPOINT="$EDUSHARING_RLP_URL/rest/lti/v13/details"
+EDUSHARING_RLP_CLIENT_ID_ON_SERLO_EDITOR=
```

Comment:
What is the reasoning behind adding them to the template? 
In environment `local` we cannot test with them sadly. 
I assumed `.env.template` is just for `local`.